shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/offers/takeOffer/PaymentMethodPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package network.bisq.mobile.presentation.ui.uicases.offers.takeOffer
+
+import kotlinx.coroutines.flow.StateFlow
+import network.bisq.mobile.domain.data.model.OfferListItem
+import network.bisq.mobile.domain.service.offerbook.OfferbookServiceFacade
+import network.bisq.mobile.presentation.BasePresenter
+import network.bisq.mobile.presentation.MainPresenter
+import network.bisq.mobile.presentation.ui.navigation.Routes
+
+// TODO: Should do Interface for this?
```

Comment:
if you didn't need it for a view then answer is probably not

Reply:
That was an old comment. Not applicable now. Sorry!
Removing it